### PR TITLE
Use node 20

### PR DESCRIPTION
**Pull request by jingsam**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

docs update

## What is the current behavior?

`app/studio/README.md` still use node 18, should be node 20.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by charislam — APPROVED** [review]
Thanks @jingsam!

**Comment by charislam** [comment]
Hey @jingsam, can you do me a favor and ping me (but don't update the branch/merge latest changes) next time you notice this hasn't merged? I wanna know what exactly is failing that has prevented this from auto-merging over and over 😢

**Comment by jingsam** [comment]
ping @charislam

**Comment by charislam** [comment]
Ahh it's the Prettier check. I tried to push a formatting commit but for some reason was denied access (even though I see that contributors can edit this branch...) Do you mind running a quick `npm run format`?

File: apps/studio/README.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ Project settings are managed outside of the Dashboard. If you use docker compose
 
 ### Developer Quickstart
 
-> [!NOTE]  
+> [!NOTE]
```
Comment (alaister) [inline apps/studio/README.md:36]:
```suggestion
> [!NOTE]   
```

File: apps/studio/README.md [hunk 2]
```diff
@@ -33,11 +33,11 @@ Project settings are managed outside of the Dashboard. If you use docker compose
...
```
Comment (alaister) [inline apps/studio/README.md:36]:
```suggestion
> [!NOTE]  
```

**Pull request merged**